tests/test_oml/test_datasets/test_datasets.py

```diff
@@ -141,3 +151,74 @@ def test_image_datasets() -> None:
     check_base(dataset_qgl)
     check_query_gallery(dataset_qgl, df_val)
     check_labeled(dataset_qgl, df_val)
+
+
+def get_df() -> pd.DataFrame:
```

Comment:
hm, you probably understood me wrong on moving tests into another py file, I'm sorry if it was not clear enough. The idea was:

`def test_audio_datasets(df: pd.DataFrame) -> None:` - keep it in `test_datasets` (as it's done now) because we don't test domain specific stuff, we just check that all audio datasets are aligned with the general interfaces

add `test_audio_datasets.py` file for domain specific stuff and put `test_downmix`, `test_resmaple_trim_pad`, `test_start_times` there in order not to mix it up with testing general functionality.

Reply:
done